public/app/plugins/datasource/loki/components/LokiOptionFields.test.tsx

```diff
@@ -3,19 +3,35 @@ import React from 'react';
 
 import { LokiOptionFieldsProps, LokiOptionFields } from './LokiOptionFields';
 
+interface Props extends LokiOptionFieldsProps {
+  queryType: string;
+  lineLimitValue: string;
+  onLineLimitChange: () => void;
+  onQueryTypeChange: () => void;
+  onKeyDownFunc: () => void;
+}
+
 const setup = (propOverrides?: LokiOptionFieldsProps) => {
   const queryType = 'range';
   const lineLimitValue = '1';
   const onLineLimitChange = jest.fn();
   const onQueryTypeChange = jest.fn();
   const onKeyDownFunc = jest.fn();
+  const resolution = propOverrides?.resolution!;
+  const query = propOverrides?.query!;
+  const onChange = propOverrides?.onChange!;
+  const onRunQuery = propOverrides?.onRunQuery!;
 
-  const props: any = {
+  const props: Props = {
     queryType,
```

Comment:
Some of the props here, that are later spreaded to the component, don't exist in `LokiOptionFieldsProps`.